### PR TITLE
Cleanup & refactor

### PR DESCRIPTION
**Pull request by baybara-pavel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Generates Swift snippets from colors, fonts and layers.
 
   ```swift
   final class RadialGradientView: UIView {
-  
+
```
Comment (artemnovichkov) [inline README.md]: Did you remove newline symbol from code snippet?

File: README.md [hunk 2]
```diff
@@ -129,13 +129,36 @@ Generates Swift snippets from colors, fonts and layers.
 
 ## How to Install
 
-The instruction from official [tutorial](https://github.com/zeplin/zeplin-extension-documentation/blob/master/tutorial.md#adding-a-local-extension).
+Download and unzip [the latest release](https://github.com/artemnovichkov/zepcode/releases/tag/0.2.0).
```
Comment (artemnovichkov) [inline README.md]: With this logic, you have to update README every release. Let's use just https://github.com/artemnovichkov/zepcode/releases

File: README.md [hunk 3]
```diff
@@ -129,13 +129,36 @@ Generates Swift snippets from colors, fonts and layers.
...
+npm i
+```
+
+To start developing run:
+
+```bash
+  npm start
+```
+
+And finaly, to make a production ready build run:
```
Comment (artemnovichkov) [inline README.md]: A typo, `finally`

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,7071 @@
+{
+  "name": "zepcode",
+  "version": "0.1.0",
```
Comment (artemnovichkov) [inline package-lock.json]: Still 0.1.0 vers

File: src/helpers.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
+function camelize(str) {
+  return str.replace(/\W+(.)/g, (match, chr) => chr.toUpperCase());
+}
+
+function uiColor(color) {
+  return `UIColor(red: ${color.r}/255, green: ${color.g}/255, blue: ${
```
Comment (artemnovichkov) [inline src/helpers.js]: Why here are newlines here?

File: README.md (hunk 1 above)
Comment (baybara-pavel) [reply README.md]: Nope, it's just my text-editor was clean not-important empty symbols and also newline still here.

File: src/helpers.js (hunk 5 above)
Comment (baybara-pavel) [reply src/helpers.js]: Line too long and it's standard prettier auto formatting

**Pull request merged**